app/src/main/java/com/chess/clock/activities/ClockTimersActivity.java

```diff
@@ -634,6 +563,66 @@ private void updateUIState() {
         clockMenu.updateSoundIcon(soundManager.areSoundsEnabled());
     }
 
+    private void onPlayerOnClockClicked() {
```

Comment:
One is misspelled as "On". That said I don't like that code duplication in click listeners, if we change anything we will have to remember and verify that it was changed twice.